server/anonymous/anonymous.go

```diff
@@ -30,17 +35,30 @@ type Config struct {
 // PluginAPI API form mattermost plugin
 type PluginAPI interface {
 	SendEphemeralPost(userID string, post *model.Post) *model.Post
+	GetActivePlugins() ([]PluginIdentifier, error)
 }
 
 type anonymous struct {
 	Config
+	verifiedPlugins map[PluginIdentifier]bool
+
+	unverifiedPluginsList []PluginIdentifier
+	unverifiedPluginsLock *sync.RWMutex // guards
```

Comment:
Since these variables are a part of the same logic structure (plugin verification) it would be better to bundle them in a separate structure like Config

Reply:
should we do the same in there?
https://github.com/bakurits/mattermost-plugin-anonymous/blob/74a697a2e491458df045b6d96d14e576e1e4243b/server/plugin/plugin.go#L39
or is it too much?

Reply:
we should do the same there